text/0517-io-os-reform.md

```diff
@@ -1370,6 +1370,91 @@ The contents of `std::io::net` submodules `tcp`, `udp`, `ip` and
 the other modules are being moved or removed and are described
 elsewhere.
 
+#### InetAddr
```

Comment:
I don't like how similar IpAddr and InetAddr are, and I think it's not obvious which one is which, just from looking at the names.

Could we choose either Inet or Ip and have, say, InetAddr (host address) and InetSocketAddr?